src/styles/layout/_contributions-browser.scss

```
@@ -0,0 +1,490 @@
+  height: 8px;
+  border-radius: 50%;
+  margin-inline-end: 0.35rem;
+}
+
+.cr-stars {
+  white-space: nowrap;
+  font-family: var(--font-mono);
+  font-size: 0.72rem;
+  color: var(--text-dim);
+
+  :global(svg) { width: 12px; height: 12px; opacity: 0.7; vertical-align: -1px; margin-inline-end: 0.2rem; }
```

Stylelint (17.12.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)